src/modules/sharing/sharing.controller.ts

```diff
@@ -1026,4 +1029,12 @@ export class SharingController {
 
     return { message: 'User removed from shared folder' };
   }
+
+  @UseFilters(new HttpExceptionFilter())
+  @UseGuards(ThrottlerGuard)
+  @Public()
+  @Get(':id/size')
```

Comment:
Just to be a little more clear that this is only for public shared folders, could you change it to something like
` @Get('/public/:sharingId/folder/size')` ? You can use any naming you consider better anyways, we just need to make clear that this is not going to return file size or private sharings size.

Reply:
it's for shared files, could be public or private

Reply:
This should not be the case. A private sharing should be protected under auth checks like a logged user, that has access (because is the `owner` or the `sharedWith` user) to the metadata explicitly.